engineering/pr-etiquette.md

```diff
@@ -2,61 +2,61 @@
 layout: default
 ---
 
-# Etiquette while raise a PR for authors
+# Pull Request Guidelines
```

Comment:
Code review tip: View the updated md file [here](https://github.com/jalantechnologies/handbook/blob/ccf22b2a97f3ea3810d5f4a9ac9a4fe89089b776/engineering/pr-etiquette.md)